spec.md

```diff
@@ -3656,8 +3656,9 @@ Together with the divide operator, the modulus operator satisfies the relationsh
 
 | Name   | Arguments | Parameters | Arg Types   | Result Type | Result Width |
 |--------|-----------|------------|-------------|-------------|--------------|
-| lt,leq |           |            | (UInt,UInt) | UInt        | 1            |
-| gt,geq | (e1,e2)   | ()         | (SInt,SInt) | UInt        | 1            |
+| lt,leq | (e1,e2)   | ()         | (UInt,UInt) | UInt        | 1            |
+| gt,geq |           |            | (SInt,SInt) | UInt        | 1            |
+| eq,neq |           |            |             |             |              |
```

Comment:
This might be more consistent (and would require formatting):

```suggestion
| lt,leq,gt,geq,eq,neq | (e1,e2)   | ()         | (UInt,UInt) | UInt        | 1            |
|                      |           |            | (SInt,SInt) | UInt        | 1            |
```

Reply:
Unfortunately, this doesn't fit in the rendered version.

Reply:
I made a compromise, grouping them in pairs. It looks passable:
<img width="974" alt="image" src="https://github.com/chipsalliance/firrtl-spec/assets/115587324/a99d66b5-9519-40ef-8cf4-930412e13d5c">